src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -63,7 +63,7 @@
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.0.11020001-fabe022e" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.0.3" />
     <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.1.7" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.23-11785" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.23" />
```

Comment:
Do test projects also need to have the updated version ? Applies to the v2.x PR too.

Reply:
Test projects do not have references to webjobs SDK!

Reply:
Ah, sorry my bad.